### ISSUE TITLE
Recommondation for more realism/behavior

### ISSUE THREAD

**Issue by VivienHardgas** [s4t4n/FS22_CVT_Addon#51]
Ive tested the mod now for a few hours and i really like the attention to detail and the realism it brings. However, i want to make a recommendation on the behavior of throttle. 

When i press the accelerate pedal, the engine speed wont go up as it should, instead the engine seems to only speed up as the pump need wich is not what it should be. 

From what i know and seen the gas pedal is more or less a gas pedal and when i press it the engine speeds up. the pump in the CVT works the with the input its gets. 
in TMS from my understanding the gearbox and other parts manage the engine speed as to thier needs like the mods does right now without TMS on.

that's what i know and understand from the technical papers and vids ive seen. 

i really would like to see something like that to be implemented, and/or why it is what it is.

So und jetzt nochmal auf Deutsch:

Ich habe die Mod jetzt ein paar Stunden getestet und der Detailgrad gefällt mir sehr.

Mir ist aber aufgefallen, dass ein paar Dinge nicht so sind, wie sie sein sollten. 

Wenn ich mit dem Trecker Gas gebe, dreht der Motor nicht hoch, sondern lässt sich eher vom Getriebe diktieren was es braucht. 

Von meinem Verständnis her ist ein solches Verhalten eher dem TMS-Modus zuzuschreiben, denn normal in echt sollte der Motor unabhängig vom Getriebe hochdrehen und das Getriebe setzt diese Kraft um. 

Der TMS-Modus regelt von dem, was ich weiß über die Signale verschiedener Steuergeräte und Getriebe weiß, selbst die Motordrehzahl.
Sprich, wenn das Getriebe mehr Leistung für die Pumpe und z. B. Zapfwelle braucht, sagt es dem Motor, dass dieser mehr Drehzahl geben muss.

Wenn der TMS-Modus aber aus ist, hat der Fahrer die volle Kontrolle über die Motordrehzahl und das Getriebe regelt den Input, aber nicht den Motor.

Ich weiß nicht, ob das umsetzbar ist oder ob das richtig ist, aber diverse Videos von z. B. 7810diaries drehen viele Schlepper u. a. Fendt teils auf 2000 u/min hoch ohne den TMS-Modus. 

In dem Sinne würde ich mich über eine Antwort freuen und mich auch gerne richtig stellen lassen, falls dies nicht stimmen sollte.

Mit freundlichen Grüßen 

Vivien Hardgas

**Comment by s4t4n** [comment]
Hi, danke für deine ausführliche Beschreibung,

das ist völlig richtig mit dem TMS und rpm, jedoch ist das nicht umsetzbar, da es keinen definierten Motor im LS für CVT's gibt, bzw. es wird gar nicht angegeben, ebenso wenig das Getriebe. Wenn ich das so umsetze, funktionieren die Schalter nicht mehr.

Dennoch wollte ich die erpm anhebe-Charaktaristik haben.

Selbiges mit der Beschl.-Rampe, in RL ist das anders herum daß es ungesund wird, lässt sich aber nicht umsetzen.
Der TMS Modus, den ich versucht habe, ist gleichermaßen nicht so ganz realisierbar, da sobald das Gaspedal betätigt wird, der LS sofort Vortrieb umsetzen möchte. RPM und Übersetzung dort zu separieren ist nicht möglich, zumindest nicht für den Multiplayer.

Mal sehen was im 25er so machbar sein wird.

Viele Grüße
SbSh

**Comment by miracleman51** [comment]
Hello,  do you have an update if you will release this for FS25?  Loved the mod in 2022!

**Comment by SbSh-DNA-ModDing** [comment]
Yes, cvt addon will come for fs25 when it's done and we'll see if it's possible to realise this feature. At FS22 it was not confirm with the gameengine.
